package.json

```diff
@@ -27,6 +27,8 @@
     "type": "git",
     "url": "https://github.com/microsoft/vscode-docker"
   },
+  "extensionKind": "ui",
```

Comment:
Do I have to do anything special to debug an extension when the kind is set to "ui"? [These instructions](https://code.visualstudio.com/api/advanced-topics/remote-extensions#debugging-your-extension-in-a-remote-environment) work for me when the kind is "workspace", but I don't see the docker explorer when it's set to "ui"